libmscore/CMakeLists.txt

```diff
@@ -63,7 +63,7 @@ add_library (
       bracket.cpp breath.cpp bsp.cpp changeMap.cpp chord.cpp chordline.cpp
       chordlist.cpp chordrest.cpp clef.cpp cleflist.cpp
       drumset.cpp durationtype.cpp dynamic.cpp dynamichairpingroup.cpp edit.cpp noteentry.cpp
-      element.cpp elementgroup.cpp excerpt.cpp
+      easeInOut.cpp element.cpp elementgroup.cpp excerpt.cpp
```

Comment:
See above

Reply:
Done. I decided on renaming the file with a lowercase 'e'

Reply:
Nope, not yet, just for the .h so far ;-)

Reply:
I think you'd still need to rename the file `EaseInOut.cpp` to `easeInOut.cpp`

Reply:
And the Linux build errors prove me right it seems

Reply:
Hmm, no, mtest and vtest failed for different reasons apparently

Reply:
It just crashes all over the place